napari/utils/settings/_manager.py

```diff
@@ -0,0 +1,177 @@
+"""Settings management.
+"""
+
+import os
+from pathlib import Path
+
+from appdirs import user_config_dir
+from pydantic import ValidationError
+from yaml import safe_dump, safe_load
+
+from napari.utils.settings._defaults import CORE_SETTINGS
```

Comment:
Should be relative import I think, unless there's a reason for this to be different?

Reply:
Fixing it now!

Reply:
Fixed!